README.md

```diff
@@ -129,20 +129,20 @@ mongoSchedule.on('debug', ({ data, message }: MomoEvent) => {
 
 ### MomoEvent and MomoErrorEvent
 
-| event | property         | type                     | description                                                                                                                                  |
-| ----- | ---------------- | ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------- |
-| both  | message          | `string`                 | Some information about the event that occurred.                                                                                              |
-| both  | data (optional)  | `{ name?: string; ... }` | Contains additional information like the name of the affected job.                                                                           |
+| event | property         | type                     | description                                                                                                                                   |
+| ----- | ---------------- | ------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| both  | message          | `string`                 | Some information about the event that occurred.                                                                                               |
+| both  | data (optional)  | `{ name?: string; ... }` | Contains additional information like the name of the affected job.                                                                            |
 | error | type             | `MomoErrorType`          | one of: `'defining job failed'`, `'scheduling job failed'`, `'executing job failed'`, `'stopping job failed'`, `'an internal error occurred'` |
-| error | error (optional) | `Error`                  | The root cause of the error.                                                                                                                 |
+| error | error (optional) | `Error`                  | The root cause of the error.                                                                                                                  |
 
 ### Job Examples
 
 ```
 const example1: MomoJob = {
   name: 'example 1',
   interval: '5 minutes',
-  delay: 0,
+  firstRunAfter: 0,
```

Comment:
I don't think it adds much to have two equivalent examples here (1 and 2). I would only have one without `firstRunAfter` and then add to the explanation that it will run immediately because the default is `0`.